amber/convert_amber.py

```diff
@@ -1247,6 +1258,68 @@ def validate_merged_lipids(ffxml_name, leaprc_name):
             os.unlink(f[1])
     if verbose: print('Lipids energy validation for %s done!' % ffxml_name)
 
+def validate_glyco_protein(ffxml_name, leaprc_name,
```

Comment:
Use this test idea?

https://github.com/ParmEd/ParmEd/pull/1149#issuecomment-782546139